telegram/bot.py

```diff
@@ -3212,8 +3228,10 @@ def pin_chat_message(self, chat_id, message_id, disable_notification=None, timeo
     @log
     def unpin_chat_message(self, chat_id, timeout=None, **kwargs):
         """
-        Use this method to unpin a message in a supergroup. The bot must be an administrator in the
-        chat for this to work and must have the appropriate admin rights.
+        Use this method to unpin a message in a group, a supergroup, or a channel.
+        The bot must be an administrator in the chat for this to work and must have the
+        ‘can_pin_messages’ admin right in the supergroup or ‘can_edit_messages’ admin right
```

Comment:
Please use "``" instead of "‘’". It renders better in sphinx

Reply:
Okay. I think I've seen those single quotes in other places too. I'll change all.